security/pfSense-pkg-snort/files/usr/local/pkg/snort/snort_check_for_rule_updates.php

```diff
@@ -447,8 +447,7 @@ function snort_copy($srcFilePathPattern, $destPath){
 
 /* Save current state (running/not running) for each enabled Snort interface */
 $active_interfaces = array();
-init_config_arr(array('installedpackages', 'snortglobal', 'rule'));
```

Comment:
It's safe to leave this init_config_arr() call if you want to retain it

Reply:
After analyzing it a bit more, I don't think the init_config_arr() call is really necessary. Utilizing the default empty array return value from the follow-on config_get_path() function suffices.